reformatting webhook into easier to read groupings

DetectiveSquirrel opened the pull request.

DetectiveSquirrel committed:
reformatting webhook into easier to read groupings

instantsc commented on DataProcessor/Methods.cs:
```diff
@@ -122,19 +122,19 @@ await using (var stream2 = oldFilePath.OpenRead())
                 foreach (var fork in @new.OrderBy(x => x.Author).ThenBy(x => x.Name))
                 {
-                    sb.AppendLine($"**New** (added) fork by __{fork.Author}__ (<{GithubUrls.Repository(fork.Location, fork.Name)}>)");
+                    sb.AppendLine($"> \n> **New** (added) fork by [{fork.Location}/{fork.Name}](<{GithubUrls.Repository(fork.Location, fork.Name)}>)");
```
Usage of Author here is deliberate, repository location doesn't always match what I want to show here.

instantsc commented on DataProcessor/Methods.cs:
```diff
@@ -122,19 +122,19 @@ await using (var stream2 = oldFilePath.OpenRead())
+                    sb.AppendLine($"> \n> **New** (added) fork by [{fork.Location}/{fork.Name}](<{GithubUrls.Repository(fork.Location, fork.Name)}>)");
                     var processedCommitMessage = SubstringBefore(fork.LatestCommit.Message, new[] { '\r', '\n' }).Replace("`", "");
-                    sb.AppendLine($"Latest commit at {fork.LatestCommit.Date.Format()} with message `{processedCommitMessage}`");
+                    sb.AppendLine($"> Latest commit at <t:{((DateTimeOffset)fork.LatestCommit.Date).ToUnixTimeSeconds()}> with message\n> `{processedCommitMessage}`");
```
Make an extension out of it or something, easier to clean up if we ever invent something less awful.

instantsc commented on DataProcessor/Methods.cs:
```diff
@@ -143,19 +143,19 @@ await using (var stream2 = oldFilePath.OpenRead())
                 foreach (var (fork, newCommit, newForkReleases) in changed.OrderBy(x => x.Fork.Author).ThenBy(x => x.Fork.Name))
                 {
-                    sb.AppendLine($"__{fork.Author}__'s fork (<{GithubUrls.Repository(fork.Location, fork.Name)}>)");
+                    sb.AppendLine($"> \n> Fork by [{fork.Location}/{fork.Name}](<{GithubUrls.Repository(fork.Location, fork.Name)}>)");
```
The author thing here as well

DetectiveSquirrel committed:
Cleanups

Changed visual on hyperlinks with the authors name rather than the location of the commit

Added wrapper for discord unix tags
